packages/react-cards/package.json

```
@@ -45,14 +45,22 @@
     "@fluentui/keyboard-key": "^0.2.13",
     "@fluentui/react-make-styles": "^9.0.0-alpha.2",
     "@fluentui/react-utilities": "^9.0.0-alpha.2",
-    "@fluentui/set-version": "^8.0.0-beta.2",
-    "@fluentui/utilities": "^8.0.0-beta.15",
+    "@fluentui/set-version": "^8.0.0",
+    "@fluentui/utilities": "^8.0.0",
     "tslib": "^1.10.0"
   },
   "peerDependencies": {
     "@types/react": ">=16.8.0 <18.0.0",
     "@types/react-dom": ">=16.8.0 <18.0.0",
     "react": ">=16.8.0 <18.0.0",
     "react-dom": ">=16.8.0 <18.0.0"
+  },
+  "beachball": {
+    "tag": "beta",
+    "disallowedChangeTypes": [
+      "major",
+      "minor",
+      "patch"
+    ]
```

```suggestion
  }
```

Can exclude this change? This package should become converged and bumped to `v9.0.0-alpha` IMO

---

I was trying to set this up so the 9-alpha bumps could be done separately. Regardless, adding this config is necessary (in this package and all the others that should remain alpha or beta) due to the [change in repo defaults](https://github.com/microsoft/fluentui/pull/17169/files#diff-d1b249ccd2c506312fe38c774d9e0ecda9319a8c790ddf34568b7b09e3d388d1)--otherwise they would start bumping patch/minor versions, which isn't what we want.